<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>690</width>
    <height>510</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>–ö–æ–Ω—Å—Ç—Ä—É–∫—Ç–æ—Ä –ñ–î –°—Ç–∞–Ω—Ü–∏–π</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="12,0">
    <item>
     <widget class="MyView" name="graphicsView">
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QToolButton" name="toolButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
        <property name="text">
         <string>üñ∞</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_6">
        <property name="text">
         <string>–û–∫–Ω–æ</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>–°—Ç–µ–Ω–∞</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>–ñ–î –ü—É—Ç—å</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_4">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>–õ–µ—Å—Ç–Ω–∏—Ü–∞</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_5">
        <property name="text">
         <string>–õ–∞—Å—Ç–∏–∫</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <item>
         <property name="text">
          <string>–ö–∞—Ñ–µ</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>–ú–µ–¥–ø—É–Ω–∫—Ç</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>–¢—É–∞–ª–µ—Ç</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>–ö–∞—Å—Å–∞</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>–û–≥–Ω–µ—Ç—É—à–∏—Ç–µ–ª—å</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>–ò–Ω—Ñ–æ—Ä–º–∞—Ü–∏–æ–Ω–Ω–∞—è –¥–æ—Å–∫–∞</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>–î–æ–±–∞–≤–∏—Ç—å</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>690</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>–ú–µ–Ω—é</string>
    </property>
    <addaction name="action_1"/>
    <addaction name="action_2"/>
    <addaction name="action_3"/>
    <addaction name="action_4"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="action">
   <property name="text">
    <string>–î–æ–±–∞–≤–∏—Ç—å –ø—Ä–µ–¥–º–µ—Ç</string>
   </property>
  </action>
  <action name="action_1">
   <property name="text">
    <string>–ó–∞–≥—Ä—É–∑–∏—Ç—å –ø–ª–∞–Ω</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>–°–æ—Ö—Ä–∞–Ω–∏—Ç—å –ø–ª–∞–Ω</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>–ù–æ–≤—ã–π –ø–ª–∞–Ω</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>–≠–∫—Å–ø–æ—Ä—Ç –≤ –∫–∞—Ä—Ç–∏–Ω–∫—É</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyView</class>
   <extends>QGraphicsView</extends>
   <header>my_view.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
